### PR TITLE
Add Examples section and a 2D Canvas Context example.

### PR DESCRIPTION
**Pull request by anssiko**
@huningxin @robman Please review. This is based on an example contributed by @robman.

(Given code examples are in <pre>s, no leading whitespace, and indentation is intentionally off.)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -398,6 +398,47 @@
...
+  // NOTE: Only the R and G bytes are set to carry 16 bits of data
+  var depthVideo = document.querySelector('#depthVideo');
+  // construct a new MediaStream out of the existing depth track(stream)
+  var depthStream = new MediaStream(stream.getDepthTracks()[0]);
+  depthVideo.srcObject = depthStream;
+  depthVideo.play();
+
+  depthVideo.onloadedmetadata = function () {
+    setInterval(function () {
+      var pixels = canvasContext.drawImage(video);
```
Comment (huningxin) [inline index.html]: It probably looks better like this:

```
canvasContext.drawImage(video);
var rgbImageData = canvasContext.getImageData(0, 0, w, h);
var pixels = rgbImageData .data;
```

File: index.html [hunk 2]
```diff
@@ -398,6 +398,47 @@
...
+      var depthPixels = canvasContext.drawImage(depthVideo);
+      var dexels = new Uint16Array();
+      // iterate through depth pixels to convert 2 bytes into 1 Uint16
+      // do things with pixels and dexels here
```
Comment (huningxin) [inline index.html:445]: We may need code sample to show how to reconstruct 16 bit depth value.

File: index.html [hunk 3]
```diff
@@ -398,6 +398,47 @@
...
+  // construct a new MediaStream out of the existing depth track(stream)
+  var depthStream = new MediaStream(stream.getDepthTracks()[0]);
+  depthVideo.srcObject = depthStream;
+  depthVideo.play();
+
+  depthVideo.onloadedmetadata = function () {
+    setInterval(function () {
+      var pixels = canvasContext.drawImage(video);
+      var depthPixels = canvasContext.drawImage(depthVideo);
+      var dexels = new Uint16Array();
```
Comment (huningxin) [inline index.html]: It probably looks better like this:

```
depthCanvasContext.drawImage(depthVideo);
var depthImageData = depthCanvasContext.getImageData(0, 0, w, h);
var dexels = depthImageData .data;

for (var x = 0; x < w ; ++x) {
  for (var y = 0; y < h; ++y) {
    var i = (x + y * w) * 4;
    var depth = dexels[i] | dexels[i + 1] << 8;
    // depth is the 16 bit depth value at (x, y)
  }
}
```

**Comment by anssiko** [comment]
@ningxin Updated and landed with your changes. Make sure it passes JSLint or http://jshint.com/. You can issue another pull request and reopen issue https://github.com/w3c/mediacapture-depth/issues/40 to fix errors if any.

**Pull request merged**